runners/direct-java/build.gradle

```diff
@@ -18,15 +18,19 @@
 
 import groovy.json.JsonOutput
 
+//the list used for defining include set and the set of project level dependencies
+def dependOnProjects = [":beam-model-pipeline", ":beam-runners-core-construction-java",
```

Comment:
Can you rebase and pick up the latest changes since there is a merge conflict now?